src/main/java/seedu/address/model/Model.java

```diff
@@ -193,51 +193,70 @@
     void appendCommandHistoryEntry(String commandText);
 
     /**
-     * Returns true if a issue with the same identity as {@code issue} exists in the address book.
+     * Returns true if a issue with the same identity as {@code issue} exists in SunRez.
+     *
+     * @param issue to be checked
+     * @return {@code True} if issue is in SunRez
+     * @throws NullPointerException if {@code issue} is null.
      */
     boolean hasIssue(Issue issue);
 
     /**
      * Deletes the given issue.
      * The issue must exist in SunRez.
+     *
+     * @param target to be deleted
+     * @throws NullPointerException if {@code target} is null
      */
     void deleteIssue(Issue target);
 
     /**
      * Adds the given issue.
+     *
+     * @param issue to be added
+     * @throws NullPointerException if {@code issue} is null
      */
     void addIssue(Issue issue);
 
     /**
      * Replaces the given Issue {@code target} with {@code editedIssue}.
-     * {@code target} must exist in the address book.
+     * {@code target} must exist in the SunRez.
```

Comment:
Missing punctuation and captialisation. Param description should be separated from param name.